.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,65 @@
+{
+  "Browser Compatibility Data Template": {
+    "scope": "json",
+    "prefix": ["bcd", "browser-compat"],
+    "body": [
+      "{",
+      "  \"${1|api,css,html,http,javascript,mathml,svg,webdriver,webextensions,xpath,xslt|}\": {",
```

Comment:
It might be a good idea to add `exslt` (see&nbsp;#2973).

Reply:
No, please see the discussion in #2973. If would decide to add exslt, it can get done in #2973.